templates/guides/quickstart-dhcp.md

```diff
@@ -0,0 +1,236 @@
+---
+page_title: "Managing DHCP service with the BloxOne Terraform Provider"
+subcategory: "Guides"
+description: |-
+  This guide provides step-by-step instructions for using the BloxOne Terraform Provider to manage IPAM and DHCP resources.
+---
+
+# Managing DHCP service with the BloxOne Terraform Provider
+
+This guide provides step-by-step instructions for using the BloxOne Terraform Provider to manage IPAM and DHCP resources.
+
+## Configuring the Provider
+
+The provider needs to be configured with an API key and the URL of the Infoblox Cloud Services Portal (CSP). The API Key can be obtained from the Infoblox Cloud Services Portal (CSP) by following the steps in the guide - [Configuring User API Keys](https://docs.infoblox.com/space/BloxOneCloud/35430405/Configuring+User+API+Keys).
+
+Create a directory for the Terraform configuration and create a file named `main.tf` with the following content:
+
+````terraform
+terraform {
+  required_providers {
+    bloxone = {
+      source  = "infobloxopen/bloxone"
+      version = "~> 0.1.0"
+    }
+  }
+}
+
+provider "bloxone" {
+  csp_url = "https://csp.infoblox.com"
+  api_key = "<BloxOne API Key>"
+}
+````
+
+!> Warning: Hard-coded credentials are not recommended in any configuration file. It is recommended to use environment variables.
+
+You can also use the following environment variables to configure the provider:
+`BLOXONE_CSP_URL` and `BLOXONE_API_KEY`.
+
+Initialize the provider by running the following command. This will download the provider and initialize the working directory.
+
+```shell
+terraform init
+```
+
+## Configuring Resources
+
+### IPAM and DHCP Resources
+In this example, we will use the following resources to create an IP space, address block, and subnet.
+
+- [bloxone_ipam_ip_space](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/resources/ipam_ip_space)
+- [bloxone_ipam_address_block](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/resources/ipam_address_block)
+- [bloxone_ipam_subnet](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/resources/ipam_subnet)
+
+Add the following to the `main.tf` file:
+
+````terraform
+// Create an IP space
+resource "bloxone_ipam_ip_space" "this" {
+  name = "example"
+}
+
+// Create an address block within the IP space
+resource "bloxone_ipam_address_block" "this" {
+  space   = bloxone_ipam_ip_space.this.id
+  address = "10.0.0.0"
+  cidr    = "16"
+}
+
+// Create a subnet within the address block
+resource "bloxone_ipam_subnet" "this" {
+  space   = bloxone_ipam_ip_space.this.id
+  address = "10.0.0.0"
+  cidr    = "24"
+}
+
+````
+
+We can now run `terraform plan` to see what resources will be created.
```

Comment:
Guides are usually written in an impersonal style. The beginning of the guid talks to the user with "You", it should continue to do so all along.

Reply:
Fixed the "we" language.